src/app/(screens)/clEvent.tsx

```diff
@@ -94,10 +95,25 @@ export default function ClEventDetail(): JSX.Element {
                       ]),
                 ...(clEvent?.location != null && clEvent?.location !== ''
                     ? [
-                          {
-                              title: t('pages.event.location'),
-                              value: clEvent?.location,
-                          },
+                          Object.assign(
+                              {
+                                  title: t('pages.event.location'),
+                                  value: clEvent?.location,
+                              },
+                              isValidRoom(clEvent.location)
+                                  ? {
+                                        onPress: () => {
+                                            router.navigate({
+                                                pathname: '/map',
+                                                params: {
+                                                    room: clEvent?.location,
+                                                },
+                                            })
+                                        },
+                                        textColor: 'blue',
```

Comment:
Please use the theme primary color instead of static blue.

Reply:
resolved in [13af99b](https://github.com/neuland-ingolstadt/neuland.app-native/pull/138/commits/13af99b679044627beca501774fff104af5ce9a8)